
## S'il y a des accents dans le nom de cavit√© qui est dans le fichier .tro cela produit une erreur

from pytro2th import tro2th

tro2th(fle_tro_fnme = 'G16-.tro')